### DISCUSSION TITLE
Change the Editor

### DISCUSSION THREAD

**Discussion by blaggacao** [outline/outline#3336] [Ideas]
Honestly, writing in outline, I regularly want to scream and beat my PC in these little moments where the wysywig editor just slightly doesn't do the thing that I want it to do.

Especially since beeing accustomed to a raw editing experience or at least a marktext.app experience.

Good news: https://github.com/marktext/muya

**Comment by tommoor** [comment]
Hi @blaggacao – if you have specific examples of places where the editor doesn't behave as expected that would be useful feedback and you're welcome to file individual issues to get bugs addressed. We won't be changing the editor out for a third party editor as it's tied to our collaborative backend and upcoming commenting system.

**Reply by blaggacao** [reply]
Hm, this is bad news. I am really constantly hit by the editor. I don't think it's in a good shape. Honest opinion.

So often, that I come to believe it's not the right to diminute it into individual issues.

There is just so many instances of where I'm stuck in the editing workflow and have to delete and restart over.

A few examples:

- links not rendering when pasting from a coll phone
- indenting code blocks when part of a list
- not being able to _italic_ in parens
- wired semantics of de-indenting (enter)

**Reply by blaggacao** [reply]
How would I even give reproducible instructions short of making a screen recording?

**Reply by tommoor** [reply]
Screen recordings are a great way to report.

**> links not rendering when pasting from a coll phone**
particularly interested in this one. Where you are pasting _from_ is a factor – every app puts different things on the clipboard.

**> not being able to italic in parens**
Fixed
